test/specs/basic/case-sensitive-diacritics.js

```diff
@@ -0,0 +1,26 @@
+/*!***************************************************
+ * mark.js
+ * https://github.com/julmot/mark.js
+ * Copyright (c) 2014–2016, Julian Motz
+ * Released under the MIT license https://git.io/vwTVl
+ *****************************************************/
+"use strict";
+describe("case senstive mark", function () {
```

Comment:
Please try to follow the `describe` description of other basic tests:
"case senstive mark" => "basic mark with with case sensitive"
This looks more beautiful in the console.